l10n/en-US/viewer.ftl

```diff
@@ -320,6 +320,8 @@ pdfjs-editor-highlight-button =
 pdfjs-editor-highlight-button-label = Highlight
 pdfjs-highlight-floating-button =
     .title = Highlight
+    .aria-label = Highlight
```

Comment:
You need a new string ID to add an attribute
https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html

Reply:
> You need a new string ID to add an attribute
> https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html

Could the Fluent linter be configured to catch that?

Reply:
Technically, that's available in compare-locales. I've never tried to integrate it in a workflow though
https://github.com/mozilla/compare-locales/blob/main/compare_locales/lint/cli.py#L21-L24